packages/admin/addon/controllers/admin.ts

```diff
@@ -0,0 +1,36 @@
+/**
+ * Copyright 2020, Yahoo Holdings Inc.
+ * Licensed under the terms of the MIT license. See accompanying LICENSE.md file for terms.
+ */
+
+import Controller from '@ember/controller';
+import { tracked } from '@glimmer/tracking';
+
+/**
+ * @constant ADMIN_TAB_OPTIONS
+ */
+const ADMIN_TAB_OPTIONS = [
+  {
+    title: 'Manage Users',
+    route: 'admin.roles'
+  }
+];
+
+export default class AdminController extends Controller {
+  /**
+   * @property {Array} tabOptions
+   */
+  @tracked tabOptions = ADMIN_TAB_OPTIONS;
+
+  /**
+   * @property {boolean} isTabExpanded
+   */
+  @tracked isTabExpanded: boolean = true;
```

Comment:
```suggestion
  @tracked isTabExpanded = true;
```
Typescript should be able to infer that this is a boolean when you assign true to it.